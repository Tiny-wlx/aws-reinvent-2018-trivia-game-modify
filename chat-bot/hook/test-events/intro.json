{
  "messageVersion": "1.0",
  "invocationSource": "DialogCodeHook",
  "userId": "ozw40htm8dvj0n1lk9ugt6gja961h4dl",
  "sessionAttributes": {

  },
  "requestAttributes": null,
  "bot": {
    "name": "TriviaGame",
    "alias": "$LATEST",
    "version": "$LATEST"
  },
  "outputDialogMode": "Text",
  "currentIntent": {
    "name": "LetsPlay",
    "slots": {
      "fifteen": null,
      "nine": null,
      "six": null,
      "one": null,
      "seven": null,
      "three": null,
      "two": null,
      "thirteen": null,
      "eight": null,
      "fourteen": null,
      "four": null,
      "sixteen": null,
      "twelve": null,
      "eleven": null,
      "ten": null,
      "five": null
    },
    "confirmationStatus": "None"
  },
  "inputTranscript": "Let's play"
}
