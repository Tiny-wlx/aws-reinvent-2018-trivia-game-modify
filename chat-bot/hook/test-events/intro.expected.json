{
  "sessionAttributes": {
    "started": true,
    "currentQuestion": 1,
    "currentSlot": "one",
    "currentScore": 0
  },
  "dialogAction": {
    "type": "ElicitSlot",
    "intentName": "LetsPlay",
    "slots": {
      "fifteen": null,
      "nine": null,
      "six": null,
      "one": null,
      "seven": null,
      "three": null,
      "two": null,
      "thirteen": null,
      "eight": null,
      "fourteen": null,
      "four": null,
      "sixteen": null,
      "twelve": null,
      "eleven": null,
      "ten": null,
      "five": null
    },
    "slotToElicit": "one",
    "message": {
      "contentType": "PlainText",
      "content": "Let's play re:Invent Trivia! The game covers four categories with four questions each. Starting with the \"Know Your History\" category, for 100 points: What year was the first AWS re:Invent held?"
    }
  }
}
